src/database/to-dos.js

```diff
@@ -1,42 +1,55 @@
-const {getDatabase} = require('./mongo');
-const {ObjectID} = require('mongodb');
+const { getDatabase } = require('./mongo');
+const { ObjectID } = require('mongodb');
+const { paginate } = require('./../helper');
 
 const collectionName = 'to-dos';
 
 async function insertToDo(toDo) {
   const database = await getDatabase();
-  const {insertedId} = await database.collection(collectionName).insertOne(toDo);
+  const { insertedId } = await database
+    .collection(collectionName)
+    .insertOne(toDo);
   return insertedId;
 }
 
-async function getToDos() {
+async function getToDos(req) {
   const database = await getDatabase();
-  return await database.collection(collectionName).find({}).toArray();
+  const perPage = parseInt(req.query.per_page ? req.query.per_page : 20);
+  const rawData = database.collection(collectionName);
+  const currentPage = parseInt(req.query.page ? req.query.page : 1);
```

Comment:
Instead of:

```js
parseInt(req.query.page ? req.query.page : 1);
```

You could do:

```js
parseInt(req.query.page || 1);
```

Same applies to line 17.